spec/spec_helper.rb

```diff
@@ -1,5 +1,24 @@
 require 'simplecov'
 require 'simplecov-console'
+require_relative './setup_test_database'
+ENV['ENVIRONMENT'] = 'test'
+
+
+# require our Sinatra app file
+require File.join(File.dirname(__FILE__), '..', 'app.rb')
+
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+
+# tell Capybarac about our app class
+Capybara.app = Chitta
```

Comment:
app.rb class name is  different from the name used in spec_helper.rb to connect with Capybara.